src/Symfony/Bundle/FrameworkBundle/Tests/CacheWarmer/ValidatorCacheWarmerTest.php

```diff
@@ -79,9 +79,8 @@ public function testWarmUpWithAnnotations()
         $values = $fallbackPool->getValues();
 
         $this->assertInternalType('array', $values);
-        $this->assertCount(2, $values);
+        $this->assertCount(1, $values);
         $this->assertArrayHasKey('Symfony.Bundle.FrameworkBundle.Tests.Fixtures.Validation.Category', $values);
-        $this->assertArrayHasKey('Symfony.Bundle.FrameworkBundle.Tests.Fixtures.Validation.SubCategory', $values);
```

Comment:
@nicolas-grekas due to moving the `array_filter` I needed to fix this test. It seems before we stored empty data in the cache?